.github/workflows/auto_build.yml

```
@@ -0,0 +1,233 @@
+    - name: Accept SDK licenses
+      run: |
+        yes | sdkmanager --licenses || true
+        sdkmanager "platform-tools" "platforms;android-34"
+
+    # Build song song cả Gradle toàn cục và Wrapper
+    - name: Build with Global Gradle
+      id: build-global
+      run: |
+        echo "Building with Global Gradle $GRADLE_VERSION"
+        gradle assemble${{ steps.detect-project.outputs.BUILD_TYPE }} --stacktrace --info
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)